### DISCUSSION TITLE
How to get the list of all events

### DISCUSSION THREAD

**Discussion by mikaoelitiana** [pretix/pretix#4619] [Q&A] [answered]
I want to build a website with multiple event organisers and list all available events in my front-page. What I see in the docs is that the events API is only available by organizer https://docs.pretix.eu/en/latest/api/resources/events.html#endpoints. I would have expected `/api/v1/events` to be available.
Would I need to loop over all organizers in order to get the full list of events? That would make it hard to perform a search and filters on events. Is there another way?

**Comment by raphaelm** [comment, marked as answer]
Sorry, there is no feature for this, as it is not part of the intended scope
